### PR TITLE
add info about disabling analytics

### PR DESCRIPTION
**Pull request by devoncarew**
- add info about disabling analytics (fix https://github.com/flutter/flutter/issues/3537)

@Hixie @sethladd @qchong

### CODE REVIEW

**Comment by sethladd** [comment]
lgtm (modular travis being sad)

Thanks!

**Comment by devoncarew** [comment]
Looks like I fixed a reference for `master` and travis is analyzing against `alpha`.

File: setup.md [hunk 1]
```diff
@@ -41,6 +41,25 @@ itself. Subsequent runs should be much faster.
...
+```
+
+And to display the current state of analytics reporting:
+
+```
+flutter config
+```
+
+See Google's privacy policy:
+[www.google.com/intl/en/policies/privacy](https://www.google.com/intl/en/policies/privacy/).
```
Comment (Hixie) [inline setup.md]: Man, you really want to put a dot at the end of that URL!

File: setup.md [hunk 2]
```diff
@@ -41,6 +41,25 @@ itself. Subsequent runs should be much faster.
...
+
```
Comment (Hixie) [inline setup.md]: Maybe explicitly state that the very first run, and any runs involving `flutter config`, are not logged.

File: setup.md (hunk 1 above)
Comment (devoncarew) [inline setup.md]: But it's totally not in the hyperlink here :) - people will end up in the right place

**Pull request merged**